src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1139,11 +1151,29 @@ def get_crash_data(crash_data, symbolize_flag=True):
 
   is_kasan = 'KASAN' in crash_stacktrace_without_inlines
   is_golang = '.go:' in crash_stacktrace_without_inlines
+  is_python = '.py", line' in crash_stacktrace_without_inlines
+  found_python_crash = False
   found_golang_crash = False
   ubsan_disabled = 'halt_on_error=0' in environment.get_value(
       'UBSAN_OPTIONS', '')
 
-  for line in crash_stacktrace_without_inlines.splitlines():
+  split_crash_stacktrace = crash_stacktrace_without_inlines.splitlines()
+
+  if is_python:
+    # Python stacktraces are reversed, and followed by an ASAN one
+    python_crash_stacktrace = []
+    for line in split_crash_stacktrace:
+      if not line:
+        continue
+      if '===========' in line:  # begining of ASAN stacktrace, skip this
+        break
+      python_crash_stacktrace.append(line)
+    # Add the "title" of the stacktrace at the top
+    split_crash_stacktrace = [python_crash_stacktrace[0]]
```

Comment:
This is slightly hacky. We can have the stdout from the program in the first line (or even first few lines). I think you should add a few stdout lines in output at start like "Line1\nLine2\n" and then in the for loop, note the index for "Uncaught Python exception" marker and then only reverse lines after that.

Reply:
+1